docs/includes/apiargs-MongoDBCollection-method-rename-param.yaml

```diff
@@ -0,0 +1,28 @@
+arg_name: param
+name: $toCollectionName
+type: string
+description: |
+  The new name of the collection. If a full namespace is not
+  specified, the database of the original collection will be used. If the new
+  namespace specifies a different database, the command copies the collection
+  to the new database and drops the source collection.
```

Comment:
This text is leftover from when we had a single parameter for the collection name or namespace. I suggest replacing it with:

> The new name of the collection.